docs/usage/models.md

```diff
@@ -871,19 +890,23 @@ print([pet for pet in pets])
 
 ## Faux immutability
 
-Models can be configured to be immutable via `allow_mutation = False`. When this is set, attempting to change the
+Models can be configured to be immutable via `frozen = True`. When this is set, attempting to change the
 values of instance attributes will raise errors. See [model config](model_config.md) for more details on `Config`.
 
+!!! note
+    Same behavior in Pydantic V1 can be achieved via `allow_mutation = False`. This config is deprecated in Pydantic V2.
```

Comment:
```suggestion
    This behavior can be achieved in Pydantic V1 via `allow_mutation = False`. This config is deprecated in Pydantic V2.
```

Reply:
Does this still communicate the intended meaning? I read this note as indicating what the equivalent behavior was in V1